src/cli/serve/integration_tests/reload_logging_config.test.js

```diff
@@ -178,10 +179,11 @@ describe('Server logging configuration', function () {
         '--config', testConfigFile,
         '--logging.dest', logPath,
         '--plugins.initialize', 'false',
-        '--logging.json', 'false'
+        '--logging.json', 'false',
+        '--verbose'
       ]);
 
-      watchFileUntil(logPath, /http server running/, 2 * minute)
+      watchFileUntil(logPath, /starting server/, 2 * minute)
```

Comment:
we need this change not to wait for a migration to complete?

Reply:
Yes, I could also skip migrations, but "starting server" get run just after we add the SIGHUP handler, so I thought we could rather speed up the test a bit.